processor/cascadingfilterprocessor/sampling/span_properties_filter_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampling
+
+import (
+	"math"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+var operationNamePattern = "foo.*"
+var minDurationMicros = int64(500)
+var minNumberOfSpans = 2
```

Comment:
Surround this in a `const` and make it a block

```suggestion
const (
	operationNamePattern = "foo.*"
	minDuration          = 500 * time.Microsecond
	minNumberOfSpans     = 2
)

```

Reply:
Unfortunately, I am dealing with pointers later on and it's either limited or ugly: https://stackoverflow.com/questions/30716354/how-do-i-do-a-literal-int64-in-go/30716481

I'll switch to `vars` though